applications/CompressiblePotentialFlowApplication/custom_processes/compute_embedded_lift_process.cpp

```diff
@@ -46,20 +46,15 @@ void ComputeEmbeddedLiftProcess<Dim, NumNodes>::Execute()
         auto it_elem=mrModelPart.ElementsBegin()+i;
         auto r_geometry = it_elem->GetGeometry();
 
-        BoundedVector<double,3> geometry_distances;
+        BoundedVector<double, NumNodes> geometry_distances;
         for(unsigned int i_node = 0; i_node<NumNodes; i_node++){
             geometry_distances[i_node] = r_geometry[i_node].GetSolutionStepValue(GEOMETRY_DISTANCE);
         }
         bool is_embedded = PotentialFlowUtilities::CheckIfElementIsCutByDistance<Dim,NumNodes>(geometry_distances);
 
         if (is_embedded && it_elem->Is(ACTIVE)){
-            array_1d<double,3> elemental_distances;
-            for(unsigned int i_node = 0; i_node<NumNodes; i_node++){
-                elemental_distances[i_node] = r_geometry[i_node].FastGetSolutionStepValue(GEOMETRY_DISTANCE);
-            }
 
-            const Vector& r_elemental_distances=elemental_distances;
-            ModifiedShapeFunctions::Pointer pModifiedShFunc = this->pGetModifiedShapeFunctions(it_elem->pGetGeometry(), r_elemental_distances);
+            ModifiedShapeFunctions::Pointer pModifiedShFunc = this->pGetModifiedShapeFunctions(it_elem->pGetGeometry(), Vector(geometry_distances));
```

Comment:
I directly use the Vector constructor now in the ModifiedShFunc call. Do you like this?